src/screens/ProfileScreen/elements/AnonymousTab.js

```diff
@@ -23,21 +23,21 @@ const styles = StyleSheet.create({
     fontSize: 14,
     lineHeight: 17,
     textAlign: 'center',
-    color: isActive ? COLORS.signed_primary : COLORS.blackgrey
+    color: isActive ? COLORS.anon_primary : COLORS.blackgrey
   }),
   anonymousPostTextSubtitle: (isActive) => ({
     fontFamily: isActive ? fonts.inter[600] : fonts.inter[400],
     fontSize: 10,
     textAlign: 'center',
-    color: isActive ? COLORS.signed_primary : COLORS.blackgrey
+    color: isActive ? COLORS.anon_primary : COLORS.blackgrey
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This change is consistent with the previous one, ensuring that the subtitle text also changes color based on the active state of the tab. Good job maintaining consistency in your UI changes.
```diff
-    color: isActive ? COLORS.signed_primary : COLORS.blackgrey
+    color: isActive ? COLORS.anon_primary : COLORS.blackgrey
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->